<ui version="4.0" >
 <class>msgWnd</class>
 <widget class="QWidget" name="msgWnd" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QWidget" native="1" name="protoWidget" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QComboBox" name="msgTypeBox" >
          <property name="focusPolicy" >
           <enum>Qt::StrongFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="recieverID" />
     </item>
     <item>
      <widget class="QToolButton" name="groupBtn" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="historyBtn" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="userInfoBtn" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="encodingBtn" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="closeWindBtn" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" native="1" name="userInfoWidget" >
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QPushButton" name="pushButton" >
        <property name="text" >
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="0" column="1" >
       <widget class="QLineEdit" name="lineEdit" />
      </item>
      <item rowspan="2" row="1" column="0" >
       <widget class="QLineEdit" name="lineEdit_2" />
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="lineEdit_3" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="msgsView" />
   </item>
   <item>
    <widget class="QWidget" native="1" name="msgWidget" >
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="pushButton_2" >
        <property name="text" >
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3" >
        <property name="text" >
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4" >
        <property name="text" >
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5" >
        <property name="text" >
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="msgEdit" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
